platform/android/java/lib/src/org/godotengine/godot/Godot.java

```diff
@@ -538,6 +538,8 @@ protected void onCreate(Bundle icicle) {
 		window.addFlags(WindowManager.LayoutParams.FLAG_TURN_SCREEN_ON);
 		mClipboard = (ClipboardManager)getSystemService(Context.CLIPBOARD_SERVICE);
 
+		requestPermissions();
```

Comment:
Move this call to the `GodotEditor.java` class.

Reply:
Isn't it the best if it happens in the main, so anyone who have permission in their app will happaned automatically?

Reply:
Not really. Some app may want to do this at start time, while others may want to request permissions only after the user performs a specific action.
In either case, the method is accessible from GDScript, so it's best to leave the decision to the app developer.

I've added https://github.com/thebestnom/godot/pull/1 against your branch which should address most of the comments I made here.